.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,3 +14,10 @@ Fixes:
 - [ ] Meaningful title, helpful description and a linked NuGet/Home issue
 - [ ] Added tests
 - [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.
+
+<!--
+Note: please make sure you follow the following guidelines.
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md
+https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md
```

Comment:
Should the contributing doc be refencing all of these instead?
Feels like it's the landing point for development, so I'd everything to be linked there and kept up to date. It's also something I've seen in other repos.

Reply:
+1 - just need a comment linking to our contributing guide.
But if you are going to try to list/maintain them here, I'd like the UI guidelines to be included.

Reply:
I think it is great points raised here. I guess https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md#resources-for-nugetclient-development should be enough then